### PR TITLE
Add pull down to reload lists

### LINKED ISSUE #75: Allow scroll on lists
**Describe the bug**
Add scroll to Lists, for example we can't really see our last Friend because he is hidden by the fixed sub-navigation.

### PR DESCRIPTION
**Pull request by vkaelin**
Made a new more simple version with this lib I saw last time: https://github.com/thmsgbrt/react-simple-pull-to-refresh

Doesn't look that bad I think.

### CODE REVIEW

**Comment by MrF3lix** [comment]
Wow it seems much better

File: src/components/list/animated-list.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { List, ListProps } from '~/components/list/list';
+import { ListItem } from '~/types/List';
+import PullToRefresh from 'react-simple-pull-to-refresh';
+
+interface AnimatedListProps<T> extends ListProps<T> {
+  reload: () => Promise<unknown>;
+}
+
+export const AnimatedList: React.FC<AnimatedListProps<ListItem>> = ({
```
Comment (MrF3lix) [inline src/components/list/animated-list.tsx]: Can we rename it to `RefreshableList` or something?

**Review by MrF3lix — DISMISSED** [review]
Some small change. And would it be possible to change the loading icon when pulling down? something a bit less prominent

**Comment by vkaelin** [comment]
Made a new version, tell me what you think @MrF3lix 
https://user-images.githubusercontent.com/33511688/198996140-d13f6e63-a97a-406b-b2d0-43e163e622a4.mov

**Comment by MrF3lix** [comment]
Perfekt, you can merge it :)

**Review by MrF3lix — APPROVED** [review]

**Pull request merged**